app/views/DR/Advices.js

```diff
@@ -3,8 +3,8 @@ import { BackHandler, ScrollView, StyleSheet, View } from 'react-native';
 
 import iconAdvertisement from '../../assets/images/idea.jpg';
 import imgAdvertisement from '../../assets/images/recommendations.jpg';
-import HeaderImage from '../../components/DR/ActionCards/HeaderImage';
-import List from '../../components/DR/ActionCards/List';
+import HeaderImage from '../../components/HeaderImage';
```

Comment:
Update path of components, these components are still in DR/ActionCards and not in DR

Reply:
Home screen is duplicated in entry.js

Reply:
The home screen I fixed it somewhere else but Its not merged yet